tests/ui/keyword/soup.rs

```diff
@@ -0,0 +1,30 @@
+//@ edition:2024
+//@ run-pass
```

Comment:
Suggestion: actually, I just noticed, can you make this a `//@ check-pass`? We're not really exercising runtime behavior here.